Misc/NEWS.d/next/Library/2025-04-18-04-47-39.gh-issue-123503.mfg0wD.rst

```diff
@@ -0,0 +1,6 @@
+Subsitute internal usages of deprecated attributes :attr:`addinfourl.code
+<urllib.response.addinfourl.code>` and :meth:`HTTPResponse.info
+<http.client.HTTPResponse.info>` for their recommended equivalents
```

Comment:
```suggestion
Replace usages of deprecated attributes :attr:`addinfourl.code
<urllib.response.addinfourl.code>` and :meth:`HTTPResponse.info
<http.client.HTTPResponse.info>` with their recommended equivalents
```

“Substitute A for B” means remove B and replace it with A.

Reply:
Ups! my bad.